### PR TITLE
#8604  docs(nimbus): Update documentation to include how to update fork with main

### PR DESCRIPTION
**Pull request by Philipotieno**
Because

- We have not specified how to updat the fork with changes on the main branch

This commit

- Updates the read me with how to add an upstream remote
- Updates the read me to include how to update the fork with the latest changes on the main repository.

### CODE REVIEW

**Comment by jrbenny35** [comment]
Hey @Philipotieno 👋🏾 , thanks for the PR!

I think I understand what you are saying but following the steps, your changes wouldn't have much of an effect. When you clone the repo, you are already getting the latest changes, so you don't need to fetch them again.

What were your thoughts when you were creating this PR?

**Comment by Philipotieno** [comment]
Hello @jrbenny35 I realized that whenever I cloned my forked  repo, I was missing the changes that have been merged on the main repo, so I had to fetch the changes from the main and merge them to my fork, I might have missed something. but for now I have to fetch changes every time for an existing fork, that's why I raised the PR.

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
Thank you @Philipotieno I think the best place to add this is [contributing.md](https://github.com/mozilla/experimenter/blob/main/contributing.md)

**Comment by Philipotieno** [comment]
Hi @yashikakhurana, I've updated the `contributing.md`

File: contributing.md [hunk 1]
```diff
@@ -199,3 +199,17 @@ If the deployment succeeds but the change inadvertently breaks stage/production,
...
+1.  Go to your cloned directory and add the main repository as your upstream
+
+        git remote add upstream  https://github.com/mozilla/experimenter
+
+1.  Fetch the latest changes from the main repository
+
+        git fetch upstream
+
+1.  Merge the changes from the main repository into your forked branch
+
```
Comment (yashikakhurana) [inline contributing.md:214]: you can probably remove 1. above from the three places, give it actual numbering like 1,2,3 or may be bullets like `-`

**Review by yashikakhurana — CHANGES_REQUESTED** [review]

**Review by yashikakhurana — APPROVED** [review]
thank you @Philipotieno

**Pull request merged**